Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj

```diff
@@ -2,9 +2,13 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <RootNamespace>Jellyfin.Plugin.OpenSubtitles</RootNamespace>
```

Comment:
What is the significance of this change?

Reply:
This was already the root namespace so not needed

Reply:
As specified in the `.sln`?

Reply:
It's specified by the project name + folders